[2025-09-15T12:36:09.535+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-15T12:36:09.589+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_model_dag_batch_iterative_85000.wait_for_predict_api manual__2025-09-15T12:36:04.007883+00:00 [queued]>
[2025-09-15T12:36:09.635+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_model_dag_batch_iterative_85000.wait_for_predict_api manual__2025-09-15T12:36:04.007883+00:00 [queued]>
[2025-09-15T12:36:09.636+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-09-15T12:36:09.691+0000] {taskinstance.py:2888} INFO - Executing <Task(HttpSensor): wait_for_predict_api> on 2025-09-15 12:36:04.007883+00:00
[2025-09-15T12:36:09.729+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=389) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-15T12:36:09.722+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'train_model_dag_batch_iterative_85000', 'wait_for_predict_api', 'manual__2025-09-15T12:36:04.007883+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmp9jal92da']
[2025-09-15T12:36:09.736+0000] {standard_task_runner.py:72} INFO - Started process 399 to run task
[2025-09-15T12:36:09.739+0000] {standard_task_runner.py:105} INFO - Job 43: Subtask wait_for_predict_api
[2025-09-15T12:36:09.991+0000] {task_command.py:467} INFO - Running <TaskInstance: train_model_dag_batch_iterative_85000.wait_for_predict_api manual__2025-09-15T12:36:04.007883+00:00 [running]> on host c34d0ef88347
[2025-09-15T12:36:10.395+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_model_dag_batch_iterative_85000' AIRFLOW_CTX_TASK_ID='wait_for_predict_api' AIRFLOW_CTX_EXECUTION_DATE='2025-09-15T12:36:04.007883+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-15T12:36:04.007883+00:00'
[2025-09-15T12:36:10.403+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-15T12:36:10.507+0000] {baseoperator.py:405} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2025-09-15T12:36:10.559+0000] {http.py:137} INFO - Poking: /health
[2025-09-15T12:36:10.563+0000] {base.py:84} INFO - Retrieving connection 'predict_api'
[2025-09-15T12:36:10.644+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-09-15T12:36:10.729+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-15T12:36:10.732+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=train_model_dag_batch_iterative_85000, task_id=wait_for_predict_api, run_id=manual__2025-09-15T12:36:04.007883+00:00, execution_date=20250915T123604, start_date=20250915T123609, end_date=20250915T123610
[2025-09-15T12:36:10.831+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-15T12:36:10.877+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
